### ISSUE TITLE
optional issues

### ISSUE THREAD

**Issue by kahsa** [vlang/v#16410]
- [ ] https://github.com/vlang/v/issues/6717
- [ ] https://github.com/vlang/v/issues/10507
- [ ] https://github.com/vlang/v/issues/10803
- [ ] https://github.com/vlang/v/issues/11293
- [ ] https://github.com/vlang/v/issues/13068
- [ ] https://github.com/vlang/v/issues/14185
- [ ] https://github.com/vlang/v/issues/14381
- [ ] https://github.com/vlang/v/issues/14492
- [ ] https://github.com/vlang/v/issues/14515
- [ ] https://github.com/vlang/v/issues/15549
- [ ] https://github.com/vlang/v/issues/15581
- [ ] https://github.com/vlang/v/issues/15781
- [ ] https://github.com/vlang/v/issues/15990
- [ ] https://github.com/vlang/v/issues/16211
- [ ] https://github.com/vlang/v/issues/16339
- [ ] https://github.com/vlang/v/issues/16407
- [ ] https://github.com/vlang/v/issues/16408
- [ ] https://github.com/vlang/v/issues/16409

**Comment by shove70** [comment]
Thanks for organizing! well done.
The problems related to the struct fields will be solved in PR #16392, please wait :)

**Comment by kahsa** [comment]
Yes, it seems so. When put them together, most of the problems are the same.
I am very much looking forward to it.

**Comment by kahsa** [comment]
By the way, in my personal opinion, I do not like the practice of closing an Issue because it is a duplicate, because it makes it difficult to understand the magnitude of the demand for the issue. 
If there are duplicates, just associate them with each other.

**Comment by dumblob** [comment]
@shove70 you could be interested in the last comments in the discussion about RFC 7 starting from https://github.com/vlang/rfcs/pull/7#issuecomment-932583818 .

**Comment by shove70** [comment]
> @shove70 you could be interested in the last comments in the discussion about RFC 7 starting from [vlang/rfcs#7 (comment)](https://github.com/vlang/rfcs/pull/7#issuecomment-932583818) .

Thanks!

**Comment by medvednikov** [comment]
Everything has been fixed :)
